manifests/tekton/tasks/base/scos-build.yaml

```diff
@@ -8,6 +8,17 @@ spec:
     description: The version repo
     type: string
   steps: 
+  - name: copy-okd-rpms
+    image: registry.ci.openshift.org/origin/4.12:artifacts
+    script: |
+      #!/usr/bin/env bash 
+      set -euxo pipefail
+
+      mkdir /workdir/rpms
+      cp /srv/repo/openshift-{clients-0.0.1-*,hyperkube-*}.x86_64.rpm /workdir/rpms
```

Comment:
I'm currently checking the workdir mount point
Getting this error when testing on operat-first

```
[scos-build : assemble] Updating metadata for 'okd-rpms'...done
[scos-build : assemble] error: Updating rpm-md repo 'okd-rpms': /workdir/rpms-repo was not found
[scos-build : assemble] error: failed to execute cmd-fetch: exit status 1

```

Reply:
I updated to /workdir/rpms-repo as I added another mountpoint

Reply:
That's a possibility I'd say, but I thin I'd also like to investigate whether we can use Tekton's workspaces for this instead of using a volume: https://tekton.dev/vault/pipelines-v0.14.3/workspaces/

Reply:
I'd add an `ls -al /workdir` before the command and verify it looks sane...

Reply:
already tried that. it looks OK afaict